TicketSwap/solar
# ExternalLink Icon update

# Description

This icon is less bold and properly centered.

## How to test

- Checkout this branch
- Go to iconography
- Confirm it looks good

## Screenshots

<img width="97" alt="image" src="https://user-images.githubusercontent.com/12168698/217836355-1f635616-8e76-4002-ad53-c535c4fbc74d.png">
